homeassistant/components/monzo/application_credentials.py

```diff
@@ -0,0 +1,15 @@
+"""application_credentials platform the Monzo integration."""
+
+from homeassistant.components.application_credentials import AuthorizationServer
+from homeassistant.core import HomeAssistant
+
+OAUTH2_AUTHORIZE = "https://auth.monzo.com"
+OAUTH2_TOKEN = "https://api.monzo.com/oauth2/token"
```

Comment:
Should this be using the constants from `.const`? 👀

Reply:
Definitely, thanks!